models/service_test.go

```diff
@@ -0,0 +1,205 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/swift-sunshine/swscore/kubernetes"
+	"github.com/swift-sunshine/swscore/models"
+
+	"k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type prometheusClientMock struct {
```

Comment:
there is inconsistency in naming. here it starts with small letter p, and a few lines below, IstioClientMock starts with a capital letter. is this intentional? what is the naming convention?

Reply:
First letter case is not just a convention, it has a real impact: upper case gives visibility out of the package.
So I guess `IstioClientMock` here should be renamed `istioClientMock`